jenkins_job_wrecker/registry.py

```diff
@@ -16,9 +16,15 @@ class Registry(object):
     registry = {}
     project_types = {}
 
+<<<<<<< HEAD
```

Comment:
Whoops, some parts of the rebase got in here :)

Reply:
Yeah I'm trying to fix it now haha. Haven't rebased too much, I'm sort of a rookie. I think I got it this time, I'm cleaning it up and I'll push again.